resources/function_help/json/display_expression

```diff
@@ -0,0 +1,39 @@
+{
+        "name": "display_expression",
+	"type": "function",
+        "description": "Returns the display expression for a given feature in a layer. If called with no parameters, it evaluates the current feature. The expression is evaluated by default.",
+        "arguments": [
+                {
+                        "arg": "feature",
+                        "optional": true,
+                        "default": "current feature",
+                        "description": "The feature which should be evaluated."
+		},
+                {
+                        "arg": "layer",
+                        "optional": true,
+                        "default": "current layer",
+                        "description": "The layer (or its id or name)."
+                },
+		{
+			"arg": "evaluate",
+                        "description": "If the expression must be evaluated. If false, the expression will be returned as a string literal only (which could potentially be later evaluated using the 'eval' function).",
+			"optional": true,
+			"default": "true"
+		}
+	],
+        "examples": [
+                {
+                        "expression": "display_expression()",
+                        "returns": "The display expression of the current feature."
+                },
+                {
+                        "expression": "display_expression($currentfeature)",
+                        "returns": "The display expression for a given feature."
```

Comment:
@Gustry this is still for the current feature -- ref maptip example

Reply:
Sorry @DelazJ ? Is there an issue?
I just spotted `$currentFeature` vs `$currentfeature`. I will fix it

Reply:
Sorry I meant it would be
> returns": "The display expression for the current feature."

Which remains a "given" feature but... Also this is how this example is explained for maptip function

Reply:
Do you mean you prefer something like https://github.com/qgis/QGIS/blob/master/resources/function_help/json/is_selected#L21
I wrote "current feature", when the feature is coming from the scope, not as parameter